src/clj/puppetlabs/services/jruby/jruby_metrics_core.clj

```diff
@@ -85,6 +88,14 @@
    :reason RequestReasonInfo
    :time schema/Num})
 
+(def BorrowMetric
+  {:reason schema/Str
+   :count schema/Int
+   :mean schema/Num
+   ;; FIXME: This metric is currently busted for reasons outlined in
+   ;;        TK-468. Find a suitable replacement or remove it.
+   :aggregate schema/Num})
```

Comment:
I would think max or maybe 99.9th percentile would be better.

Reply:
My current thought is to drop aggregate altogether and use `:count` as the sorting key. The upstream Dropwizard Metrics library has added a sum to Timers which looks like it  will ship in version 5.x:

https://github.com/dropwizard/metrics/pull/1254

Once that lands, could add `:sum` here and to other places that report data from Timers. Then eventually deprecate and drop the `:aggregate` entries (somehow? Not sure what the best deprecation strategy would be for this sort of data.)

Percentiles would be great too and by bundling them in we could get something close to the Prometheus Summary metric:

https://prometheus.io/docs/concepts/metric_types/#summary

@npwalker @suckatrash @abottchen @jarretlavallee Any thoughts on which percentile values would be the most helpful?

Reply:
Dropped `aggregate` and added 99.9th percentile as `max` as it has the same "past five minutes" weighting as the `mean`. Also added the FiveMinuteRate as `rate` to give a measurement of request rate.